resources/ext/cli/config-migration

```diff
@@ -0,0 +1,243 @@
+#!/bin/bash
+
+migration_command="puppetdb config-migration"
+
+#############
+# FUNCTIONS #
+#############
+
+# Backs up a file, if it hasn't already been backed up
+#
+# $1 - file to backup
```

Comment:
I'd expand this into a clearer usage statement. Here's an example, but feel free to disregard it:

```shell
config-migration.sh takes one argument:
config-migration <path to file to backup>
```